spec/graphql/schema/input_object_spec.rb

```diff
@@ -297,19 +297,30 @@ class TestInput2 < GraphQL::Schema::InputObject
       TestInput2.to_graphql
     end
 
-    it "returns a symbolized, aliased, ruby keyword style hash" do
+    before do
       arg_values = {a: 1, b: 2, c: { d: 3, e: 4, instrumentId: "Instrument/Drum Kit"}}
 
-      input_object = InputObjectToHTest::TestInput2.new(
+      @input_object = InputObjectToHTest::TestInput2.new(
         arg_values,
         context: OpenStruct.new(schema: Jazz::Schema),
         defaults_used: Set.new
       )
+    end
+
+    describe "#to_h" do
+      it "returns a symbolized, aliased, ruby keyword style hash" do
+        assert_equal({ a: 1, b: 2, input_object: { d: 3, e: 4, instrument: Jazz::Models::Instrument.new("Drum Kit", "PERCUSSION") } }, @input_object.to_h)
+      end
+    end
 
-      assert_equal({ a: 1, b: 2, input_object: { d: 3, e: 4, instrument: Jazz::Models::Instrument.new("Drum Kit", "PERCUSSION") } }, input_object.to_h)
+    describe "#to_hash" do
+      it "returns the same results as #to_h (aliased)" do
+        assert_equal(@input_object.to_h, @input_object.to_hash)
+      end
     end
   end
 
+
```

Comment:
```suggestion
```

Reply:
(I tried to commit it myself but it wouldn't let me 🤷‍♂ )

Reply:
I'll try to get Rubocop working to reduce the amount of inconsistencies, thanks!